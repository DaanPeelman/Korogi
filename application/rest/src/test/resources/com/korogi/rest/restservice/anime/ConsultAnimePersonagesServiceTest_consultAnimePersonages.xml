<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <ANIME  ID="1"
            ANIME_TYPE="TV"
            NAME_EN="Steins;Gate"
            NAME_RO="Steins;Gate"
            START_AIR="2011-04-06"
            END_AIR="2011-09-14"
            SYNOPSIS="Steins;Gate synopsis here"
            BACKDROP_URL="http://backdrop.url.be/steins.gate"
            POSTER_URL="http://poster.url.be/steins.gate"

            CREATION_DATE="2018-08-01 00:00:00"
            CREATED_BY="john.doe"
            MODIFICATION_DATE="2018-01-01 00:00:00"
            MODIFIED_BY="john.doe"
            VERSION="1" />

    <PERSONAGES ID="1"
                FIRST_NAME="Okabe"
                LAST_NAME="Rintarou"
                PHOTO_URL="http://photo.url.be/okabe.rintarou"

                CREATION_DATE="2018-01-01 00:00:00"
                CREATED_BY="john.doe"
                MODIFICATION_DATE="2018-01-01 00:00:00"
                MODIFIED_BY="john.doe"
                VERSION="1" />

    <ANIME_PERSONAGES   ANIME_ID="1"
                        PERSONAGE_ID="1" />

    <PERSONAGES ID="2"
                FIRST_NAME="Makise"
                LAST_NAME="Kurisu"
                PHOTO_URL="http://photo.url.be/makise.kurisu"

                CREATION_DATE="2018-01-01 00:00:00"
                CREATED_BY="john.doe"
                MODIFICATION_DATE="2018-01-01 00:00:00"
                MODIFIED_BY="john.doe"
                VERSION="1" />

    <ANIME_PERSONAGES   ANIME_ID="1"
                        PERSONAGE_ID="2" />

    <ANIME  ID="2"
            ANIME_TYPE="TV"
            NAME_EN="Naruto"
            NAME_RO="Naruto"
            START_AIR="2002-10-03"
            END_AIR="2007-02-08"
            SYNOPSIS="Naruto synopsis here"
            BACKDROP_URL="http://backdrop.url.be/naruto"
            POSTER_URL="http://poster.url.be/naruto"

            CREATION_DATE="2018-08-01 00:00:00"
            CREATED_BY="john.doe"
            MODIFICATION_DATE="2018-08-01 00:00:00"
            MODIFIED_BY="john.doe"
            VERSION="1" />

    <PERSONAGES ID="3"
                FIRST_NAME="Naruto"
                LAST_NAME="Uzumaki"
                PHOTO_URL="http://photo.url.be/naruto.uzumaki"

                CREATION_DATE="2018-01-01 00:00:00"
                CREATED_BY="john.doe"
                MODIFICATION_DATE="2018-01-01 00:00:00"
                MODIFIED_BY="john.doe"
                VERSION="1" />
</dataset>