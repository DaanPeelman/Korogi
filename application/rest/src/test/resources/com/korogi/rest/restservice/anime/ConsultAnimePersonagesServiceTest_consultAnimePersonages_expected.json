{
  "links": [],
  "content": [
    {
      "firstName": "Okabe",
      "lastName": "Rintarou",
      "photoUrl": "http://photo.url.be/okabe.rintarou",
      "type": "personage",
      "links": [
        {
          "rel": "self",
          "href": "http://localhost/personages/1"
        }
      ]
    },
    {
      "firstName": "Makise",
      "lastName": "Kurisu",
      "photoUrl": "http://photo.url.be/makise.kurisu",
      "type": "personage",
      "links": [
        {
          "rel": "self",
          "href": "http://localhost/personages/2"
        }
      ]
    }
  ],
  "page": {
    "size": 2,
    "totalElements": 20,
    "totalPages": 10,
    "number": 1
  }
}